package.json

```diff
@@ -17,8 +17,19 @@
     "eslint": "^9.6.0",
     "globals": "^15.8.0",
     "husky": "^9.0.11",
+    "lint-staged": "^15.2.7",
     "parcel": "2.12",
     "prettier": "^3.3.2",
     "sass": "^1.77.6"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
```

Comment:
why using --write here? Do you want it to automatically format on commit?

Reply:
Yes, as it is just formatting the code, the logic will not get affected so it is more efficient to format the code automatically rather than doing it manually